packages/react-vapor/src/components/table-hoc/TableHOCUtils.ts

```diff
@@ -69,4 +76,5 @@ export const TableHOCUtils = {
     getCompositeState,
     getPredicateId,
     getPaginationId,
+    getTableIdFromPredicateId,

```

Comment:
I know there is no test file for the TableHOCUtils but I think those are worth testing. If you don't feel like writing unit tests for all the other existing and untested utils you can just write tests for `getTableIdFromPredicateId` for now, although it would be nice if they were all tested :smile:.

Reply:
No problem!